README.md

```diff
@@ -1 +1,70 @@
 # Work In Progress - Onfido Java Library
+
+The official Java library for integrating with the Onfido API.
+
+:warning: Under Construction :warning:
+
+Documentation can be found at <https://documentation.onfido.com>
+
+## Installation
+
+_Coming soon_
+
+## Getting Started
+
+Import the `Onfido.java` object, this is the main object used for interfacing with the API:
```

Comment:
I would leave `.java` off of all these class names, I think just the class name is fine? Same with the classes below